### DISCUSSION TITLE
How to debug old card likes gfx803?

### DISCUSSION THREAD

**Discussion by xuhuisheng** [ROCm/ROCm#1838] [Q&A] [answered]
Glad to see there is a space to communicate with ROCm team.

I am a RX580 user. I know ROCm cannot provide additional resources for older card. But it is very thankful if there is some exists tools or documents for debug old card, like gfx803.

For some background informations, Recently, I try to let blender-3 support gfx803. The cycles plugin of blender3 just used HIP to support software path tracing. It can work well on vega10, RDNA1, RDNA2.

I can compile successfully for AMDGPU_TARGET=gfx803, but some kernel cannot get properly result, likes the ray won't hit any triangle so it always be blank on the exported image.

The latest version of kernel driver just skip when there is not a SOC15. so gfx803 cannot use debug. But I want to ask how we debug gfx803 before it had been deprecated, I think maybe ROCm-3.5? Maybe there is some exists tools can help me to do some debug on gfx803.

I don't want to waste resources for AMD, but if AMD could provide more informations, Maybe we could maintain old card by community.

Any helps will be appriciated, Thank you very much.

**Comment by tcgu-amd** [comment, marked as answer]
Hi @xuhuisheng, our documentation support only goes back to 5.0 (see [here](https://rocm.docs.amd.com/en/latest/release/versions.html)), so unfortunately we cannot offer any official guidance at the moment. However, there might be some community resources available to help address you situation.

**Comment by timchenggu123** [comment]
Maybe try https://web.archive.org/web/20201109033350/https://rocmdocs.amd.com/en/latest/?
